packages/edit-site/src/hooks/commands/use-edit-mode-commands.js

```diff
@@ -106,10 +117,91 @@ function useEditModeCommandLoader() {
 	};
 }
 
+function useEditUICommandLoader() {
+	const { openGeneralSidebar, switchEditorMode } =
+		useDispatch( editSiteStore );
+	const { canvasMode, editorMode } = useSelect(
+		( select ) => ( {
+			isPage: select( editSiteStore ).isPage(),
+			hasPageContentFocus: select( editSiteStore ).hasPageContentFocus(),
+			canvasMode: unlock( select( editSiteStore ) ).getCanvasMode(),
+			editorMode: select( editSiteStore ).getEditorMode(),
+		} ),
+		[]
+	);
+	const { toggle } = useDispatch( preferencesStore );
+
+	if ( canvasMode !== 'edit' ) {
+		return { isLoading: true, commands: [] };
+	}
+
+	const commands = [];
+
+	commands.push( {
```

Comment:
I think it's better to `getActiveComplementaryArea` and add some of these commands conditionally.